frontend/src/components/Footer/Footer.js

```diff
@@ -30,7 +30,17 @@ const Footer = () => {
           >
             <i className="fa fa-youtube-play" aria-hidden="true"></i>
           </a>
+          <hr className={styles.divider}></hr>
+          <a
+            href="https://instagram.com/wooteco?igshid=YmMyMTA2M2Y="
+            target="_blank"
+            rel="noreferrer noopener"
+            aria-label="Instagram"
+          >
+            <i class="fa fa-instagram" aria-hidden="true"></i>
```

Comment:
```suggestion
            <i class="fa fa-instagram" aria-hidden="true" />
```

기존 코드에서도 self closing tag를 사용하지 않은 상태였네요.
몇 개의 파일을 선별해서 찾아 본 결과, self closing tag에 대한 명확한 기준이 없는 것 같네요. 

이 상황에서는 저는 self closing tag로 바꾸는 게 좋다고 생각합니다. 

그 이유는 다음과 같습니다.

1. self closing tag가 아니면, 해당 tag의 자식에 값이 들어가는 것인지 착각이 들 수 있다고 생각합니다.
2. 불필요하게 타이핑을 한다고 생각합니다.

소피아는 어떻게 생각하시나요? 🤔

Reply:
저도 록바와 같은 이유로 self closing tag를 선호하는 편입니다.
기존 코드의 i 태그와 같은 형식을 맞추기 위해 사용하지 않고 작성했었어요.

우선 Footer 내에서는 모두 self closing tag를 사용하도록 수정해봤습니다~ https://github.com/woowacourse/service-apply/pull/607/commits/057dc304220d5c0f919cf4a312d00dab775fff4b

Reply:
확인했습니다. 👍

Reply:
a: self closing tag가 저도 의미상 명확하다고 생각합니다 👍 [요런 lint 룰](https://github.com/jsx-eslint/eslint-plugin-react/blob/master/docs/rules/self-closing-comp.md)도 적용할 수 있는데요, 이렇게 맞춰가면 좋을 것 같은 부분들에 하나씩 lint 룰을 추가해나가도 좋을 것 같아요. 컨벤션을 맞추긴 하겠지만, 새로운 개발자는 알기 어렵고 문서화를 하더라도 놓치기 쉬우니까요 🙂